### ISSUE TITLE
Storyshots not working with addWithJSX

### ISSUE THREAD

**Issue by denchen** [storybooks/addon-jsx#35]
I can't seem to get Storyshots to work with this addon. The error is as follows when running `npm test`:

    FAIL  src/__tests__/storyshots.test.js
    ● Test suite failed to run
    TypeError: (0 , _react2.storiesOf)(...).addWithJSX is not a function

      at Object.<anonymous> (src/__stories__/test.stories.js:12:40)
      at node_modules/@storybook/addon-storyshots/dist/require_context.js:39:24
          at Array.forEach (<anonymous>)
      at requireModules (node_modules/@storybook/addon-storyshots/dist/require_context.js:34:9)
      at node_modules/@storybook/addon-storyshots/dist/require_context.js:48:7
          at Array.forEach (<anonymous>)
      at requireModules (node_modules/@storybook/addon-storyshots/dist/require_context.js:34:9)
      at Function.newRequire.context (node_modules/@storybook/addon-storyshots/dist/require_context.js:90:5)
      at evalmachine.<anonymous>:23:19
      at runWithRequireContext (node_modules/@storybook/addon-storyshots/dist/require_context.js:102:3)
      at testStorySnapshots (node_modules/@storybook/addon-storyshots/dist/index.js:120:35)
      at Object.<anonymous> (src/__tests__/storyshots.test.js:24:31)

Storyshots works just fine if I use Storybook's `add()` instead of `addWithJSX()`.

The only configuration I have related to `addon-jsx` is in `.storybook/config.js`:

    import { setOptions } from '@storybook/addon-options';
    import JSXAddon from 'storybook-addon-jsx';
    setAddon(JSXAddon);
    configure(loadStories, module);

And storyshots configured in `storyshots.test.js` as:

    import initStoryshots from '@storybook/addon-storyshots';
    initStoryshots();

And this is my sample story:

    import * as React from 'react';
    import { storiesOf } from '@storybook/react';
    storiesOf('test', module).addWithJSX('test1', () => <div>hi</div>);

**Comment by ruyasan** [comment]
Ditto, exact same problem. 

I was able to work around it by manually requiring `.storybook/config` without running `configure()` (by way of hacky global flag - this is definitely a work-around).

I'm sure `configPath` is being set correctly - i tried setting it explicitly (no change) and also setting it to an incorrect path to make sure it's actually being used (it is, the expected error comes up). 

It just seems like something about it's setting up storybooks is different.

**Comment by pimlottc-gov** [comment]
Also had this problem, after taking @ruyasan pointers I just copied parts of my `.storybook/config` into `storyshots.test.js`, ending up with this:

```
import initStoryshots from "@storybook/addon-storyshots";
import { setAddon } from "@storybook/react";
import JSXAddon from "storybook-addon-jsx";

setAddon(JSXAddon);

initStoryshots({
  /* configuration options */
});
```

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in v7.1.2 :rocket:
